components/Form/Autocomplete/Autocomplete.module.scss

```diff
@@ -16,27 +17,44 @@
 }
 
 .suggestionList {
-  display: block;
+  box-sizing: border-box;
+  background: lbh-colour('lbh-white');
+  border: 2px solid lbh-colour('lbh-text');
   position: absolute;
-  border-left: 1px solid #aaa;
-  border-right: 1px solid #aaa;
-  background: #fff;
-  font-size: 1.1875rem;
+  top: 14px;
+  left: 0px;
+  width: 100%;
+  list-style: none;
+  z-index: 2;
   max-height: 300px;
   overflow-y: auto;
-  border-bottom-left-radius: 4px;
-  border-bottom-right-radius: 4px;
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
-  z-index: 1;
-  width: 414px;
+  // display: block;
+  // position: absolute;
+  // border-left: 1px solid #aaa;
+  // border-right: 1px solid #aaa;
+  // background: #fff;
+  // font-size: 1.1875rem;
+  // max-height: 300px;
+  // overflow-y: auto;
+  // border-bottom-left-radius: 4px;
+  // border-bottom-right-radius: 4px;
+  // list-style-type: none;
+  // margin: 0;
+  // padding: 0;
+  // z-index: 1;
+  // width: 414px;
```

Comment:
Can we delete this commented out section

Reply:
done